### ISSUE TITLE
Check for services

### ISSUE THREAD

**Issue by EdwardAckerman** [fireworkweb/fwd#112]
I think it would be nice if before starting, fwd checked for possible services eg: `chromedriver` and it output a message asking if that service should also be run with fwd start

https://github.com/fireworkweb/fwd/blob/61c8e4186a58b96ac44781d280cd2d18279a1668/app/Tasks/Start.php#L26

**Comment by fabriciojs** [comment]
I don't think we want that, as the idea specially for `chromedriver` is that we want some services specified on docker-compose.yml but not always to run, so bothering the developer every time about what services they want to get up, don't seems like a good way to go about it, not worthy trade-off.

@EdwardAckerman what is your line of thought on this? I mean, what use cases/experience you think would improve?

**Comment by EdwardAckerman** [comment]
Makes sense

**Comment by EdwardAckerman** [comment]
we have a command to start all services already, that already covers up this right

**Comment by fabriciojs** [comment]
@EdwardAckerman I see, well in that case I am going to close this one, and in case you have any other use case that comes up and might benefit from changes on this area, please open up a new ticket.
